### ISSUE TITLE
afFieldValueIs not working consistently (broken by 5.4.1 release!)

### ISSUE THREAD

**Issue by wesyah234** [Meteor-Community-Packages/meteor-autoform#1163]
I have something like this:

```
        {{> afQuickField  name='medicalHistory.field1'}}
        {{#if afFieldValueIs name="medicalHistory.field1" value="Yes"}}
        {{> afQuickField  name='medicalHistory.field2'}}
        {{> afQuickField  name='medicalHistory.field3'}}
        {{/if}}
```

Which should show field2 and field 3 only if the value of field 1 is Yes, however, intermittently, when I open the form, if the value of field1 is Yes, it doesn't show the field2 and field 3.  If I close and reopen the form again it sometimes works properly.

the field1 is a radio button with values of Yes and No, so I can toggle between Yes and No and I see field2 and 3 appearing and disappearing properly, it's just the issue when the value is set to Yes, and sometimes the other fields are hidden, and sometimes they are not.

How should I attempt to troubleshoot this issue?

EDIT: this may be part of the issue, but the fields in question are on a form that is on a tabbed type interface using bootstrap, and the tab is not active when first coming to the page:

```
 <ul class="nav nav-tabs" role="tablist">
```

etc...

**Comment by wesyah234** [comment]
Please see this repo:
https://github.com/wesyah234/quickFormIssue
it will illustrate the problem.
also I posted on the forums: 
https://forums.meteor.com/t/problem-or-bug-with-autoforms-affieldvalueis-helper/8977
thank you for looking at this.

**Comment by tothand** [comment]
Maybe it is related to this issue https://github.com/aldeed/meteor-autoform/issues/861

I know it's an old one but seems to be very similar.

**Comment by wesyah234** [comment]
Yes. I've looked at some of those other issues, and agree.  I also opened another issue: https://github.com/aldeed/meteor-autoform/issues/1051 which was subsequently closed... as duplicate/fixed   At the time I logged #1051  i was reporting that I could get the correct behavior by switching to regular select box instead of radio button, however at this time, with my current issue mentioned above, switching to regular select box does not work.

**Comment by wesyah234** [comment]
just checked my current list of packages for the demo 
```
aldeed:autoform           5.4.1  Easily create forms with automatic insert and update, and automatic reactive validation.
aldeed:collection2        2.4.0  Automatic validation of insert and update operations on the client and server.
iron:router               1.0.9  Routing specifically designed for Meteor
meteor-platform           1.2.2  Include a standard set of Meteor packages in your app
meteorhacks:subs-manager  1.6.0  Subscriptions Manager for Meteor
mrt:moment                2.8.1  Moment.js, a JavaScript date library for dates, packaged for Meteor. See http://momentjs.com.
msavin:mongol             1.1.5  The insanely handy development package for Meteor.
twbs:bootstrap            3.3.5  The most popular front-end framework for developing responsive, mobile first projects on the web.
```

**Comment by wesyah234** [comment]
I cross-posted this to the forums, and serkandurusoy pointed out that this was a bug in the latest 5.4.1 release.  I confirmed that using 5.4.0 fixes this issue.

https://forums.meteor.com/t/problem-or-bug-with-autoforms-affieldvalueis-helper/8977

**Comment by wesyah234** [comment]
related to 
https://github.com/aldeed/meteor-autoform/issues/1136

**Comment by tcastelli** [comment]
+1 having the same problem

**Comment by solace** [comment]
I am experiencing the same issues, and reverting to 5.4.0 resolves it.

**Comment by joaobarcia** [comment]
+1

**Comment by aldeed** [comment]
I've merged some PRs. Please see whether this is fixed for you with 5.6.0 release
